tests/console/zypper_info.pm

```diff
@@ -54,12 +54,11 @@ sub run {
         return;
     }
 
-    prepare_source_repo;
-
     # check for zypper info
     test_package_output;
 
     if (is_sle('>=12-SP2') || is_opensuse) {
+        prepare_source_repo;
```

Comment:
I wonder if we should remove or at least disable sources repo afterwards...

Reply:
Pending verification run: http://artemis.suse.de/tests/363

Reply:
Pending verification run for `zypper_info` in extra_tests: http://artemis.suse.de/tests/378